src/attestation/Attester.chain.spec.ts

```diff
@@ -57,14 +57,14 @@ describe('Test Attester on chain', () => {
     }).addFromMnemonic(mnemonic)
     expect(attesterAddress).toBe(fromMiniSecret)
     expect(attesterAddress).toBe(fromMnemonic)
-    expect(attesterAddress).not.toBe(fromSeed)
```

Comment:
Seems like they have [closed the compatibility with older versions](https://github.com/polkadot-js/common/blob/master/packages/keyring/src/keyring.ts#L168) or fixed it. Anyway, an address should be the same no matter whether it was generated from the seed, mnemonic or URI. The test was designed this way only to be aware of when they fix this.